Authentication/SignUp/Controller/SignupControllerDelegate.swift

```diff
@@ -142,7 +142,7 @@ public protocol SignupControllerDelegate {  //swiftlint:disable:this class_deleg
     
 }
 
-extension SignupControllerDelegate {
+public extension SignupControllerDelegate {
```

Comment:
If the user wants to override the delegate by creating his, if they want to use the default behaviour of `shouldDisplaySignupErrorWithAlert ` shouldn't the extension where the default behaviour is defined be public?